<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add</class>
 <widget class="QDialog" name="add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="shapeBox">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>20</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Line</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polyline</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Polygon</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Rectangle</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ellipse</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Text</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Shape:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PenColor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PenWidth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PenStyle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PenCapStyle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PenJoinStyle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BrushColor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BrushStyle:</string>
   </property>
  </widget>
  <widget class="QLabel" name="textStringLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>101</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextString:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="penColorBox">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>magenta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="penWidth">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QComboBox" name="penStyle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NoPen</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SolidLine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DashLine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DotLine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DashDotLine</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DashDotDotLine </string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="penCapStyle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>FlatCap</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SquareCap</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RoundCap</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="updateButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QComboBox" name="penJoinStyle">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MiterJoin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BevelJoin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RoundJoin</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="brushColorBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>white</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>black</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>red</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>green</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>blue</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>cyan</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>magenta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>yellow</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="brushStyleBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>300</y>
     <width>111</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SolidPattern</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HorPattern</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VerPattern</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NoBrush</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CrossPattern</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="textString">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>211</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="textSizeLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text Size:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="textSize">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>42</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="textALabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text Alignment:</string>
   </property>
  </widget>
  <widget class="QLabel" name="textFontLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text Font :</string>
   </property>
  </widget>
  <widget class="QLabel" name="textSLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text Style :</string>
   </property>
  </widget>
  <widget class="QLabel" name="textWLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>81</width>
     <height>20</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Text Weight :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="textA">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>161</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>AlignLeft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AlignRight</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AlignJustify</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AlignCenter</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="textFont">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Helvetica</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Impact</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Courier</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OldEnglish</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>System</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="textStyle">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Italic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Oblique</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="textWeight">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>121</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Thin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Light</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Normal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Medium</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bold</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ExtraBold</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Black</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
